docs/tutorial/using-selenium-and-webdriver.md

```diff
@@ -86,12 +86,12 @@ const driver = new webdriver.Builder()
   // The "9515" is the port opened by chrome driver.
   .usingServer('http://localhost:9515')
   .withCapabilities({
-    chromeOptions: {
+    'goog:chromeOptions': {
```

Comment:
does `goog:chromeOptions` work with older versions too?

Reply:
Yes, I tested in the older selenium-webdriver 3.4.0 and 3.6.0 versions and `goog:chromeOptions` works.